make-dev-genesis-valid

```diff
@@ -0,0 +1,68 @@
+#!/nix/store/506nnycf7nk22x7n07mjjjl2g8nifpda-bash-4.4-p23/bin/bash
```

Comment:
Can we make all these scripts writeScriptBin so they can go into buildInputs in a nix-shell? Also if there's a good reason not to, we shouldn't hard code paths to bash in nix store